### PR TITLE
Added Free label to all the courses which are not premium

### LINKED ISSUE #164: Add free label to free section of trial course in My Course Activity

### PR DESCRIPTION
**Pull request by shubhamofbce**
Fixes #164 

Changes: Added free label to all the courses which are not premium

Screenshots for the change:
all set now

### CODE REVIEW

**Comment by aggarwalpulkit596** [comment]
Please add ss for if section title is more than one line or if it takes up all the space

**Comment by shubhamofbce** [comment]
I cant understand what are you trying to say??? What is ss

**Comment by aggarwalpulkit596** [comment]
Screenshot

**Comment by aggarwalpulkit596** [comment]
Nevermind

File: app/src/main/res/layout/item_section.xml [hunk 1]
```diff
@@ -22,17 +22,30 @@
 
             <TextView
                 android:id="@+id/title"
-                android:layout_width="wrap_content"
+                android:layout_width="276dp"
```
Comment (aggarwalpulkit596) [inline app/src/main/res/layout/item_section.xml]: Why a fixed size ?

File: app/src/main/res/layout/item_section.xml [hunk 2]
```diff
@@ -22,17 +22,30 @@
...
                 android:layout_height="wrap_content"
                 android:layout_alignParentStart="true"
                 android:layout_marginStart="10dp"
                 android:layout_marginTop="8dp"
-                android:layout_toStartOf="@id/arrow"
+                android:layout_marginEnd="43dp"
```
Comment (aggarwalpulkit596) [inline app/src/main/res/layout/item_section.xml]: 43 dp a little too much

File: app/src/main/java/com/codingblocks/cbonlineapp/adapters/SectionDetailsAdapter.kt [hunk 3]
```diff
@@ -141,12 +142,17 @@ class SectionDetailsAdapter(private var sectionData: ArrayList<CourseSection>?,
                     val hour = duration / (1000 * 60 * 60) % 24
                     val minute = duration / (1000 * 60) % 60
 
-                    if (minute >= 1 && hour == 0L)
+                    if (minute >= 1 && hour == 0L){
                         itemView.lectureTime.text = ("$minute Mins")
+                        itemView.free.text = ("  FREE  ")
```
Comment (aggarwalpulkit596) [inline app/src/main/java/com/codingblocks/cbonlineapp/adapters/SectionDetailsAdapter.kt]: Text wil always be free so just change visibility here and add this text to textview

File: app/src/main/res/layout/item_section.xml (hunk 1 above)
Comment (shubhamofbce) [reply app/src/main/res/layout/item_section.xml]: Should i make it wrap_content

File: app/src/main/res/layout/item_section.xml (hunk 2 above)
Comment (shubhamofbce) [reply app/src/main/res/layout/item_section.xml]: How much do you want??

File: app/src/main/res/layout/item_section.xml (hunk 1 above)
Comment (aggarwalpulkit596) [reply app/src/main/res/layout/item_section.xml]: wrap content match parent whatever is better

File: app/src/main/res/layout/item_section.xml (hunk 2 above)
Comment (aggarwalpulkit596) [reply app/src/main/res/layout/item_section.xml]: 16 would be fine

File: app/src/main/java/com/codingblocks/cbonlineapp/adapters/SectionDetailsAdapter.kt (hunk 3 above)
Comment (shubhamofbce) [reply app/src/main/java/com/codingblocks/cbonlineapp/adapters/SectionDetailsAdapter.kt]: I don't know the way to keep it there in the Textview and print that text for a particular condition and not to print for some other so i kept it here. 
Can u suggest me how can i do this by keeping the text in the TextView

File: app/src/main/res/layout/item_section.xml (hunk 2 above)
Comment (shubhamofbce) [reply app/src/main/res/layout/item_section.xml]: Ok

File: app/src/main/res/layout/item_section.xml (hunk 1 above)
Comment (shubhamofbce) [reply app/src/main/res/layout/item_section.xml]: Ok let me solve this

File: app/src/main/java/com/codingblocks/cbonlineapp/adapters/SectionDetailsAdapter.kt (hunk 3 above)
Comment (aggarwalpulkit596) [reply app/src/main/java/com/codingblocks/cbonlineapp/adapters/SectionDetailsAdapter.kt]: Change visibility of textview to gone ? It would do the work

**Comment by shubhamofbce** [comment]
Please review #187 i fixed all th things you mentioned in the comment

**Pull request closed without merging**